components/layout_thread/lib.rs

```diff
@@ -865,6 +879,7 @@ impl LayoutThread {
             self.webrender_api.clone_sender(),
             self.webrender_document,
             info.paint_time_metrics,
+            self.busy.clone(),
```

Comment:
We should be passing an AtomicBoolean that is stored in LayoutThreadInit, rather than cloning the one that is used by the current layout thread.

Reply:
My previous comment here is still valid - there's now a member of LayoutThreadInit that we should be passing here instead.